### DISCUSSION TITLE
Metric labels with double underscore is exposing with single underscore

### DISCUSSION THREAD

**Discussion by hiralm18** [open-telemetry/opentelemetry-collector-contrib#32372] [Q&A]
We are using windows perf counter receiver and have "perfmon_process__processor_time" configured as the name of metric. But through Open telemetry collector it converts double underscores into single underscores while finally scraping the metrics which results as "perfmon_process_processor_time". Please advise on the same.

**Comment by sameerjoshinice** [comment]
Hello Team,
I am facing the same issue where the double underscores in the name are being replaced with single underscores.
For example giving the following in the windowsperfcounters metrics level and mentioning the same name in the "metric" list at counters level , does not keep the value as double underscores and replaces it with  single underscores.
